app/src/main/java/com/example/themealsapp/di/NetworkModule.kt

```diff
@@ -1,37 +1,27 @@
 package com.example.themealsapp.di
 
-import com.example.themealsapp.rest.MealsAPI
-import com.squareup.moshi.Moshi
-import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import com.example.themealsapp.data.remote.MealsAPI
 import dagger.Module
 import dagger.Provides
 import dagger.hilt.InstallIn
 import dagger.hilt.components.SingletonComponent
 import okhttp3.OkHttpClient
 import okhttp3.logging.HttpLoggingInterceptor
 import retrofit2.Retrofit
-import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.converter.gson.GsonConverterFactory
 import java.util.concurrent.TimeUnit
 
 @Module
-@InstallIn (SingletonComponent::class)
+@InstallIn(SingletonComponent::class)
 class NetworkModule {
 
-    @Provides
-    fun providesMealsService(
-        retrofit: Retrofit
-    ): MealsAPI{
-        return retrofit.create(MealsAPI::class.java)
-    }
-
     @Provides
     fun providesRetrofit(
         okHttpClient: OkHttpClient,
-        moshi: Moshi
     ): Retrofit {
         return Retrofit.Builder()
             .baseUrl(MealsAPI.BASE_URL)
-            .addConverterFactory(MoshiConverterFactory.create(moshi))
+            .addConverterFactory(GsonConverterFactory.create())
```

Comment:
Why. did you decide to move to GSON? is there any advantage to using GSON? or disadvantage to using Moshi?

Reply:
Even thougn Gson has a noticeable slower parsing time, they all about same for medium tests, but we do not know how big the database can be, in long response tests, surprisingly Moshi has the worst performance of the bunch.

Reply:
Good to know. I'll keep that in mind for future projects

Reply:
what is the speed parsing of gson vs moshi?